src/test/java/ru/ayaz/UserTest.java

```diff
@@ -0,0 +1,19 @@
+package ru.ayaz;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertTrue;
+import static org.mockito.Mockito.mock;
+
+public class UserTest {
+
+    @Test
+    public void shouldContainVideoInListWhenVideoAdded() {
```

Comment:
&#x1F34E; **Issue**
I would seek a better (more domain specific) description for such a test.
User "ContainVideoInList" - which list ? What does "contain" mean ?
May be user owns a video. Or he just has a list of watched videos. Or he just keeps track of them. Or what ?